cmd/admin-policy-info.go

```diff
@@ -18,25 +18,34 @@
 package cmd
 
 import (
+	"os"
+
 	"github.com/fatih/color"
 	"github.com/minio/cli"
 	"github.com/minio/madmin-go"
 	"github.com/minio/mc/pkg/probe"
 	"github.com/minio/pkg/console"
 )
 
+var policyInfoFlags = []cli.Flag{
+	cli.StringFlag{
+		Name:  "policy-file, f",
```

Comment:
`output-file, o` or `save, s` are better in my opinion.

Reply:
This is an option to save just policy definition, not the metadata that is output to stdout